src/Components/Notifications/NotificationsList.tsx

```diff
@@ -350,7 +350,25 @@ export default function NotificationsList({
     })
       .then((res) => {
         if (res && res.data) {
-          setData(res.data.results);
+          const toChangeId = [
+            "PATIENT_CONSULTATION_UPDATED",
+            "PATIENT_CONSULTATION_UPDATE_CREATED",
+            "PATIENT_CONSULTATION_UPDATE_UPDATED",
+          ];
+          const modifiedData = res.data.results.map((notification: any) => {
+            if (toChangeId.includes(notification.event)) {
+              return {
+                ...notification,
+                message: notification.message.replace(
+                  "Consultation",
+                  "Log Update",
+                ),
+              };
+            } else {
+              return notification;
+            }
+          });
```

Comment:
Could you clarify what exactly this is doing? Instead of replacing the message content keys from 'Consultation' to 'Log Update', it should be done in backend.

Also, `PATIENT_CONSULTATION_UPDATED` event is not related to log updates.

Reply:
It checks if the id matches any of this 
           ``` "PATIENT_CONSULTATION_UPDATED",
            "PATIENT_CONSULTATION_UPDATE_CREATED",
            "PATIENT_CONSULTATION_UPDATE_UPDATED",```
if it does it changes the message to replace the Consultation to Log Update if it doesn't it just returns the original notification, this can also be changed in the backend
```api/v1/notification```This is the endpoint that returns the notifications you can change the message field in this